operators/grpc_operators/protos/holoscan.proto

```diff
@@ -0,0 +1,61 @@
+// SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights
+// reserved. SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package holoscan.entity;
+
+service Entity {
```

Comment:
Would it be possible to rename the package and/or definitions for clarity? Current definitions lend themselves to `holoscan_entity_Entity.Entity` and similar namings, with each "entity" having a slightly different meaning. I'm not sure what would be best here.

Reply:
I took the name from GXF's streaming plug-in and I think they used the name since it's structure is very close to a GXF Entity.